src/mixins/canvas_events.mixin.js

```diff
@@ -835,7 +872,7 @@
      */
     _fireEnterLeaveEvents: function(target, e) {
       this.fireSyntheticInOutEvents(target, e, {
-        targetName: '_draggedoverTarget',
+        oldTarget: '_draggedoverTarget',
```

Comment:
this is still half way between new and old.

Reply:
updated in this commit: https://github.com/fabricjs/fabric.js/pull/6013/commits/0cd41961d47a39153bb322bbcf725fc839ed77ac